src/cli/commands/test/mod.rs

```diff
@@ -92,6 +94,111 @@ impl From<(String, TestStatus)> for TestResult {
 	}
 }
 
+/// structure used to store a Mocked function in a cairo test entrypoint
+/// fn_name: the name of the function to mock in the cairo file
+/// fn_args: the list of parameters to replace the mock function call
+#[derive(Debug)]
+pub struct MockEntry<'a> {
+	pub fn_name: &'a str,
+	pub fn_args: &'a str,
+}
+
+/// subtype MockedFn store all the mocked functions in a single test entrypoint
+/// in a cairo file
+type MockedFn<'a> = HashMap<&'a str, Vec<MockEntry<'a>>>;
```

Comment:
I'm not sure we need them organized by entrypoint.
The way we are going to use them is by reading the `compiled.json` and parse those functions data. We will do all of those at once.